portals/edsc/config.json

```diff
@@ -9,7 +9,7 @@
     }
   },
   "footer": {
-    "attributionText": "NASA Official: Stephen Berrick",
+    "attributionText": "NASA Official: Doug Newman",
```

Comment:
Don't put a name here. The value will be set during deployments and we don't want another PR to change this name again later